File: software-engineering-concepts/Writing-Readable-Maintainable-Code/Java/Writing_Readable_and_Maintainable_Code/src/main/java/m3naming/WebHttpClient.java
package m3naming;

import org.apache.http.HttpResponse;
import org.apache.http.client.HttpClient;
import org.apache.http.client.methods.HttpGet;
import org.apache.http.impl.client.CloseableHttpClient;
import org.apache.http.impl.client.HttpClientBuilder;

import java.io.IOException;

/**
 * A WebHttpClient
 */
public class WebHttpClient {

    private HttpClient client = getDefaultClient();

    public HttpResponse sendGet(String s) throws IOException {
        return client.execute(new HttpGet(s));
    }

    private CloseableHttpClient getDefaultClient(){
        return HttpClientBuilder.create()
                .build();
    }
}

File: software-engineering-concepts/Writing-Readable-Maintainable-Code/PHP/Writing_Readable_and_Maintainable_Code/01Naming/Solution/WebHttpClient.php
<?php

class WebHttpClient
{
  private $client;

  public function __construct()
  {
    //  $this->client = new self(); 
  }
  public function sendGetRequest(string $s)
  {

    $httpResponse = $this->client->sendHttp($s);
    return $httpResponse;
  }
}
